test/e2e/e2e_test.go

```diff
@@ -23,6 +23,7 @@ import (
 )
 
 func init() {
+	testing.Init()
```

Comment:
what does this do?

Reply:
https://golang.org/doc/go1.13

```
Testing flags are now registered in the new Init function, which is invoked by the generated main function for the test. As a result, testing flags are now only registered when running a test binary, and packages that call flag.Parse during package initialization may cause tests to fail.
```

(without calling this function we get errors running the e2e tests)